module1/prime_number.rb

```diff
@@ -0,0 +1,18 @@
+#Q1. WAP to display number is prime or not
+
+
+puts "Enter a number : "
+number =gets.chomp.to_i     #input from user
+count = 0
+it = 1
+while it<=number
+    if number%it == 0
+        count+=1            #increasing count if number is divisible by it
+    end
+    it+=1   
+end
+if count == 2               #prime number have only 2 factors 1 and that number itself
```

Comment:
No need to check for all number from 2 to number, if the number is divided by any of number from 2 to (number-1) then it is not a prime.
Your logic is also correct but will require more time as we are going in loop for n times. Instead of this, we can start loop
from 2 to (number-1) and set a flag if user input number  gets completely divided by any of the number from (2 to n-1)